src/Features/CSharp/Portable/MakeLocalFunctionStatic/MakeLocalFunctionStaticCodeFixProvider.cs

```diff
@@ -44,7 +44,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                     localFunction,
                     (current, generator) => generator.WithModifiers(
                         current,
-                        generator.GetModifiers(current).WithIsStatic(true)));
+                        generator.GetModifiers(current).WithIsStatic(true)).WithTriviaFrom(localFunction));
```

Comment:
it feels like this s a bug in WithModifiers itself.  If i *anyone* adds the 'static' modifier to a local function, using the generator, i think they would expect leading trivia to be preserved.

Reply:
Ah, good point. I'll look again to see if there is a way to resolve this without explicitly copying the trivia.

Reply:
Note: you may have to copy the trivia.  It's just that that copying should likely happen inside the generator.  Indeed, i think it's supposed to do that.  But it may be the case that it was never updated to handle the local-function case.  Thanks!